docs/_embed/automatic_account_creation.md

```diff
@@ -25,6 +17,18 @@ Following account creation the end user will be sent an email to let them know a
 
 <a class="button" style="background-color:#20272B;color:white" href="https://zapier.typeform.com/to/OlPloIcW/?utm_source=zapier_marketing_website&utm_medium=embed_experience&utm_campaign=solutions_marketing_page">Enable this for your product</a>
 
+## Example Implementation
+
+The below video shows the signup flow on Google's embed experience without automatic account creation. To sign up, users are required to manually provide first and last name, email address, and go through a multi-step onboarding survey before arriving at the Zap Editor:
```

Comment:
I don't know much if there are any particular accessibility guidelines for videos, but if there's anything I can update/add to improve it, I'm all ears!! I at least tried adding a bit of a description of the video above each in case the videos couldn't be played or heard.

Reply:
yeah it's best if you can provide a transcript of the video or at least captioning, but I don't have experience with how to provide either of those